### PR TITLE
likes and comments update 12 01 19

### PR DESCRIPTION
**Pull request by tjdalsans**
displayed likes and comments for the post using "justify content: space-around" from "flex"

image

### CODE REVIEW

File: src/posts/ReactionComments.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class Comments extends React.Component {
+    
+    render() {
+      return (
+        <p>
```
Comment (sangyum) [inline src/posts/ReactionComments.js:7]: Please use Materia UI Typography component here

File: src/posts/ReactionSangs.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class Sangs extends React.Component {
+    
+    render() {
+      return (
+        <p>
```
Comment (sangyum) [inline src/posts/ReactionSangs.js:7]: Use Material UI Typography component here

File: src/posts/reactions.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.reactions {
+    display: flex;
+    justify-content: space-around;
+}
```
Comment (sangyum) [inline src/posts/reactions.css:4]: Let's add a line feed here

File: src/stories/4-Reactions.stories.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Reactions from '../posts/Reactions'
+
+export default {
+    title: 'Reactions',
+  };
+
+const sangs = 'sangs 100'
+const comments = 'comments 100'
+export const defaultRendering = () => <Reactions sangs={sangs} comments={comments} />;
```
Comment (sangyum) [inline src/stories/4-Reactions.stories.js:10]: line feed

File: src/stories/2-Posts.stories.js [hunk 5]
```diff
@@ -10,7 +10,8 @@ export default {
 const post = {
     photoUrl: 'http://writingexercises.co.uk/images2/randomimage/slimy.jpg',
     caption: 'Cat is cute',
-    publishedDate: new Date()
+    publishedDate: new Date(),
+    reactions: '1'
```
Comment (sangyum) [inline src/stories/2-Posts.stories.js:14]: Replace this with "likes" and "comments"

**Review by sangyum — CHANGES_REQUESTED** [review]
See my comments

**Review by opmanual — APPROVED** [review]

**Pull request closed without merging**